.pathplanner/settings.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
This robot width and robot length fields takes into account of the bumper width along with the track width used in the kinematics. This is so that the robot is sized correctly in the pathplanner UI.

Reply:
Where in the Path Planner docs say "robotWidth": 0.71 is a measure from bumper to bumper? Take a look at Path Planner HolonomicDriveController docs. The controller is setup with  driveBaseRadius (the radius of the drive base in meters). Docs says "for swerve drive, this is the distance from the center of the robot to the modules (the wheels)". See https://mjansen4857.com/pathplanner/docs/java/com/pathplanner/lib/controllers/PPHolonomicDriveController.html